CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+Contributions are welcomed! 
+
+When contributing to this repository, please first discuss the change you wish to make via GitHub
+issue before making a change.  This saves everyone from wasted effort in the event that the proposed
+changes need some adjustment before they are ready for submission.
+
+## Pull Request Process
+
+1. Fork the repo, push your commits to a branch of your fork, open the PR.
+2. Make sure you update the README.md where relevant.
+3. You may merge the Pull Request in once you have the sign-off of two other developers.  If you 
```

Comment:
Should we require sign-off from two developers? I think one maintainer (not developer) is enough for trivial changes.